src/Prima/Pyxis/Link/Example.elm

```diff
@@ -57,13 +59,18 @@ view model =
 appBody : Model -> List (Html Msg)
 appBody model =
     [ Helpers.pyxisStyle
-    , [ Link.simple "Visit Google" "https://www.google.it"
-      , Link.withIcon "Call us" "https://www.google.it" Link.iconPhone
-      , Link.standalone "Visit Google" "https://www.google.it"
-      ]
-        |> List.map Link.render
-        |> List.intersperse Helpers.spacer
-        |> wrapper
+    , Link.simple
+        "Visit Google"
+        "https://www.google.it"
```

Comment:
In questo caso l'`href` non deve essere obbligatorio.
Direi quindi di creare 4 funzioni di configurazione:
- `simple : String -> String -> Label`
- `simpleWithOnClick : String -> msg -> Label`
- `standalone : String -> String -> Label`
- `standaloneWithOnClick : String -> String -> Label`

In maniera da evitare di dover dedurre l'intento dell'utente facendo dei controlli sul valore di `href`.

Reply:
nei casi onClick non serve passare la string dell'href a questo punto

- `simpleWithOnClick : Label -> msg`
- `standaloneWithOnClick : Label -> msg`

Reply:
Esattamente, ho scritto male io :)